docker/Dockerfile-pytorch-deepspeech

```diff
@@ -35,7 +35,7 @@ RUN /opt/conda/bin/pip install --no-cache-dir \
 FROM armory-base AS armory-local
 RUN echo "Building Armory from local source"
 WORKDIR /armory-repo/
-COPY setup.py LICENSE MANIFEST.in README.md requirements.txt test-requirements.txt armory-base-requirements.txt .
+COPY setup.py LICENSE MANIFEST.in README.md requirements.txt test-requirements.txt armory-base-requirements.txt /armory-repo/
 COPY pytest.ini .
```

Comment:
If you want to be explicit, don't we want to do it here as well (lines 39-41

Reply:
I was looking for the minimal commit. I agree so have made another commit . It also changes `/foo/.` to `/foo/` because  the dot is even worse than the path abuse that docker imposed with `dir/` instead of `dir`.